org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/TextEditConverter.java

```diff
@@ -56,6 +58,12 @@ public TextEditConverter(ICompilationUnit unit, TextEdit edit) {
 		return converted;
 	}
 
+	public TextDocumentEdit convertToTextDocumentEdit(String uri, int version) {
```

Comment:
Why do we have to pass in the uri, if the TextEditConverter contains a CU?

Reply:
you are right, fixed